src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +484,20 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    int rows = m_pPlaylistTableModel->rowCount();
```

Comment:
I'm not sure how to write this -- BasePlaylistFeature is a base class and I don't know how to say "make a copy of the implementation class".  TODO